### PR TITLE
[FEATURE] Search results displaying and filtering

### PR DESCRIPTION
**Pull request by Benerous**
[User Story](https://github.com/ita-social-projects/GreenCity/issues/572)

Features implemented:
- getting search query and search category from popup search component (via URL queries)
image

- 'eco news' category is selected by default

- displaying found items on page
image

- sorting found items by 3 sorting types: Relevance, Newest, Oldest
image

- infinite scroll until all found pages will be displayed (9 items per one page)
image

- localization of component
image

### CODE REVIEW

**Review by skrebetsnataliia — APPROVED** [review]

File: debug.log [hunk 1]
```diff
@@ -1 +1,3 @@
 [1028/120014.369:ERROR:directory_reader_win.cc(43)] FindFirstFile: ������� �� ������� ����� ��������� ����. (0x3)
+[1115/122056.566:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1115/124235.273:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (luchward) [inline debug.log:3]: What did you add debug.log file for? I must be ignored by git.

**Review by luchward — APPROVED** [review]

File: debug.log (hunk 1 above)
Comment (Benerous) [reply debug.log:3]: Added ***.log** to **.gitignore**. Not sure if it will fix the issue.

**Review by skrebetsnataliia — APPROVED** [review]

**Review by JimHolman1199 — COMMENTED** [review]
I'm not sure so it's worth asking someone else, but what do you think about changing the searchQuery to just "q" or 'search' ?
https://moz.com/blog/the-ultimate-guide-to-the-google-search-parameters

**Comment by JimHolman1199** [comment]
please resolve conflicts )

**Review by JimHolman1199 — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]

**Review by romanovskyroma — APPROVED** [review]

**Review by romanovskyroma — APPROVED** [review]

**Pull request merged**